src/main/resources/flows-config.yaml

```
@@ -215,8 +215,10 @@ flow:
       - name: householdAdditionalIncome
   householdAdditionalIncome:
     nextScreens:
-      - name: householdMoneyOnHand
-        condition: ProgramsIncludeSnap
+      - name: householdAdditionalIncomeAmount
+  householdAdditionalIncomeAmount:
+    beforeSaveAction: ClearUnusedAdditionalIncomeFields
+    nextScreens:
       - name: expensesSignPost
```

```suggestion
      - name: householdMoneyOnHand
```

Small change but I believe that the screen after householdAdditionalIncomeAmount is the householdMoneyOnHand page.

---

Sure, I'll tweak this.

---

I change this and also added a condition for whether or not to show the amounts page.